model_deployment/README.md

```diff
@@ -19,15 +19,15 @@
  ```
     pip3 install virtualenv 
     virtualenv venv
-    C:\> venv\Scripts\activate.bat
+    venv\Scripts\activate.bat
  ```
  
```

Comment:
Please add command to deactivate the virtualenv in all three methods. The command for that is `deactivate`

Reply:
@pra17dod Is it necessary?